features/documents_widget.feature

```diff
@@ -50,7 +50,6 @@ Feature: Adding the document widget to personalisable pages
   Scenario: Adding a "Documents" widget to the my page
     Given I am on the My page personalization page
     # Safeguard to ensure the page is loaded
-    And I should see "My page block"
```

Comment:
There is an explanatory comment above:

1. delete the comment if the line is not needed anymore
2. is the line not needed anymore?